apps/server-asset-sg/prisma/schema.prisma

```diff
@@ -641,6 +644,7 @@ model AssetUser {
   lang               String
   oidcId             String
   AssetUserFavourite AssetUserFavourite[]
+  workgroups         WorkgroupsOnUsers[]
 
   @@map("asset_user")
```

Comment:
The `AssetUser.role` field can be removed, and a new field `is_admin` can be added.

Reply:
For now, i am keeping the role column in the user table, unitl we have refacotred the access logic in the frontend. Will be reomved before merging into develop (note i merge into the collection branch workgroups here)